gh-123789: `secrets.randbits` returns only non-negative int (#123801)
python/cpython: Doc/library/secrets.rst

.. function:: randbits(k)

>    Returns a non-negative int with *k* random bits.

Reviewer: ```suggestion
   Return a non-negative int with *k* random bits.
```
All functions in the Python docs try to say "return" instead of "returns". Don't look at me - Guido started that decades ago :wink:.

Author: 👀👀look

>    Returns a non-negative Python integer with *k* random bits.

Reviewer 2: ```suggestion
   Returns a non-negative int with *k* random bits.
```
nit: I think it's clearer to use the class name "int" instead of the phrase "Python integer"

Reviewer 3: Should we amend the docs there as well then: [docs.python.org/3/library/random.html#random.getrandbits](https://docs.python.org/3/library/random.html#random.getrandbits)?

>    Return an int with *k* random bits, generating non-negative integers only.

Reviewer 3: ```suggestion
   Returns a non-negative Python integer with *k* random bits.
```

Just taking the same formulation as https://docs.python.org/3/library/random.html#random.getrandbits.

Author: Thanks
